difflint/lint.py

```diff
@@ -0,0 +1,62 @@
+import os.path
+from pkg_resources import resource_filename
+import shutil
+import subprocess
+
+
+def check_linters():
```

Comment:
nit: I feel like this method name would be more intuitive if it more directly reflected the nature of its return value.